bin/cdxgen.js

```diff
@@ -411,6 +411,10 @@ const options = Object.assign({}, args, {
   noBabel: args.noBabel || args.babel === false,
   project: args.projectId,
   deep: args.deep || args.evidence,
+  output:
+    isSecureMode && args.output === "bom.json"
```

Comment:
Should we do this always or only in secure mode. Currently, the `bom.json` will be created in the current working directory.